lib/mempool.go

```diff
@@ -1463,21 +1464,33 @@ func ComputeTransactionMetadata(txn *MsgDeSoTxn, utxoView *UtxoView, blockHash *
 		realTxMeta := txn.TxnMeta.(*NFTBidMetadata)
 		_ = realTxMeta
 
-		txnMeta.NFTBidTxindexMetadata = &NFTBidTxindexMetadata{
-			NFTPostHashHex: hex.EncodeToString(realTxMeta.NFTPostHash[:]),
-			SerialNumber:   realTxMeta.SerialNumber,
-			BidAmountNanos: realTxMeta.BidAmountNanos,
-		}
+		isBuyNow := false
 
 		// We don't send notifications for standing offers.
 		if realTxMeta.SerialNumber != 0 {
 			nftKey := MakeNFTKey(realTxMeta.NFTPostHash, realTxMeta.SerialNumber)
 			nftEntry := utxoView.GetNFTEntryForNFTKey(&nftKey)
+
+			bidderPKID := utxoView.GetPKIDForPublicKey(txn.PublicKey)
+			if reflect.DeepEqual(nftEntry.OwnerPKID, bidderPKID) {
```

Comment:
Hmm.. I think this is wrong in the case where someone bids on their own non-buy-now NFT. I think an easy way to do this is to maybe use the UtxoOps of the connected txn to determine this, since that's the most fool-proof.